library/ziti.c

```diff
@@ -325,13 +335,18 @@ static void session_cb(ziti_session *session, ziti_error *err, void *ctx) {
     int errCode = err ? code_to_error(err->code) : ZITI_OK;
 
     if (init_req != NULL) {
-        init_req->nf->session = session;
+        if (session) {
+            init_req->nf->session = session;
+            ZITI_LOG(INFO, "logged in successfully => api_session[%s]", session->id);
```

Comment:
should session id be logged at all? seems like maybe a security leak?

Reply:
token is the key, can't use id to login